### DISCUSSION TITLE
Change a .tiff multi strip image to a single strip image?

### DISCUSSION THREAD

**Discussion by labSupport** [libvips/libvips#2039] [Q&A]
My problem is after compression a .tiff image with jpeg compression and then create a pdf with pdflib the log from pdflib says `No passthrough mode for TIFF image 'image.tiff' (multi-strip with JPEG compression`.

Can i change a .tiff multi strip image to a single strip image?

Do y have any idea to solve this?

**Comment by jcupitt** [comment]
Hello @labSupport,

Sorry, libvips has no control to let you set strip size in a TIFF. You could use `tiffcp` to change it, eg.:

```
tiffcp -r 5000 k2.tif x.tif
```

**Comment by labSupport** [comment]
Ok, thank you.
